.gitignore

```diff
@@ -0,0 +1,36 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+.yarn/install-state.gz
```

Comment:
It's possible to operate on wrongly formatted numbers without decimal part
![image](https://github.com/user-attachments/assets/4fecefc2-2554-41db-93e0-c1293a20943d)